scripts/data.py

```diff
@@ -0,0 +1,16 @@
+import pandas as pd
+
+"""
```

Comment:
AutoGluon common module has a nice [util](https://github.com/autogluon/autogluon/blob/master/common/src/autogluon/common/loaders/load_pd.py) to load dataframe both locally and from s3. Consider using that instead